pkg/appStore/deployment/repository/InstalledAppVersionHistory.go

```diff
@@ -26,12 +28,14 @@ func NewInstalledAppVersionHistoryRepositoryImpl(Logger *zap.SugaredLogger, dbCo
 }
 
 type InstalledAppVersionHistory struct {
-	TableName             struct{} `sql:"installed_app_version_history" pg:",discard_unknown_columns"`
-	Id                    int      `sql:"id,pk"`
-	InstalledAppVersionId int      `sql:"installed_app_version_id,notnull"`
-	ValuesYamlRaw         string   `sql:"values_yaml_raw"`
-	Status                string   `sql:"status"`
-	GitHash               string   `sql:"git_hash"`
+	TableName             struct{}  `sql:"installed_app_version_history" pg:",discard_unknown_columns"`
+	Id                    int       `sql:"id,pk"`
+	InstalledAppVersionId int       `sql:"installed_app_version_id,notnull"`
+	ValuesYamlRaw         string    `sql:"values_yaml_raw"`
+	Status                string    `sql:"status"`
+	GitHash               string    `sql:"git_hash"`
+	StartedOn             time.Time `sql:"started_on,type:timestamptz"`
+	FinishedOn            time.Time `sql:"finished_on,type:timestamptz"`
```

Comment:
consider already exists entries, ui should not be broken by this.

Reply:
okay, have already tested this scenario, will check once again for previously deployed helm apps